turbo-txpool/src/tx.rs

```diff
@@ -92,7 +112,167 @@ mod tests {
     use super::*;
 
     #[test]
-    fn tx_decode() -> Result<(), Error> {
+    fn encode_deploy() {
+        let input = vec![
```

Comment:
Thoughts on using hex strings here (in tests in general) instead?

Reply:
Something like [`hex-literal`](https://crates.io/crates/hex-literal)?

Reply:
Yeah that looks perfect. I've found that sometimes it's really nice to be able to easily copy the test cases and actually use with in the program.